main.py

```diff
@@ -152,12 +152,16 @@ def save_passwords():
 
 
 # Function to load passwords from a JSON file 
-def load_passwords():
-     """
-    Load passwords from a file into the password vault.
-
-    This function should load passwords, websites, and usernames from a text
-    file named "vault.txt" (or a more generic name) and populate the respective lists.
+def load_passwords(file_path="vault.json"):
```

Comment:
vault.json is probably a better filename than vault.txt, but need to change that in save_passwords() as well (on row 137)